src/components/@molecules/BurnFuses/BurnFusesContent.tsx

```diff
@@ -0,0 +1,276 @@
+import isEqual from 'lodash/isEqual'
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { css } from 'styled-components'
+
+import type { NamedFusesToBurn } from '@ensdomains/ensjs'
+import { Button, FlameBurnedSVG, FlameSVG, Helper, Typography } from '@ensdomains/thorin'
+
+import { Spacer } from '@app/components/@atoms/Spacer'
+import mq from '@app/mediaQuery'
+import { FuseObj } from '@app/types'
+
+export const defaultFuseObj = {
```

Comment:
deduplicate this with: `src/components/pages/profile/[name]/registration/steps/Profile/fuses.ts`